addons/sys_core/fnc_getCompartment.sqf

```diff
@@ -66,7 +67,7 @@ if (_vehicle != _unit) then {
         };
     };
 };
-if (!(typeName _compartment == "STRING")) then {
+if !(_compartment isEqualType "") then {
```

Comment:
No idea why that is used anywhere, this is shorter and to the point, will change the other files.